awx/main/models/inventory.py

```diff
@@ -1,58 +1,45 @@
 # Copyright (c) 2015 Ansible, Inc.
 # All Rights Reserved.
 
+import copy
```

Comment:
this one is actually also a part of stdlib

Reply:
Thank you! I think this was one isort pulled in. This has ben reverted here https://github.com/ansible/awx/pull/15537/commits/3c969f8d796bde9492b1da93bbeffc0d0ba5c3ab

Reply:
I meant that it was weird that it ended up above the comment saying `# Python`. It was in the module before the formatting, just elsewhere.